### PR TITLE
Updated language use in cbmc-unwinding.md

### PR DESCRIPTION
**Pull request by edstenson**
- [X] Each commit message has a non-empty body, explaining why the change was made.
- [X] NOT RELEVANT Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [X] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [X] NOT RELEVANT Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [X] NOT RELEVANT My commit message includes data points confirming performance improvements (if claimed).
- [X] My PR is restricted to a single feature or bugfix.
- [X] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: doc/cprover-manual/cbmc-unwinding.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Iteration-based Unwinding
 
-The basic idea of CBMC is to model the computation of the programs up to
+The basic idea of CBMC is to model the computation of the programs down to
 a particular depth. Technically, this is achieved by a process that
```
Comment (kroening) [inline doc/cprover-manual/cbmc-unwinding.md]: I would say that the change above is un-idiomatic.

Comment (edstenson) [reply doc/cprover-manual/cbmc-unwinding.md]: I know what you mean. My original edit was simply to change from 'up to' to 'to'. I added the down at the last minute. The basic issue here is that you can't go up to a depth.

Comment (edstenson) [reply doc/cprover-manual/cbmc-unwinding.md]: Would you be ok with just 'to'?

**Review by allredj — COMMENTED** [review]
:no_entry_sign:
This PR failed Diffblue compatibility checks (cbmc commit: c930c9ceabd23bec90f98792d904395b05817f58).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/93708712
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

The incompatibility may have been introduced by an earlier PR. In that case merging this
PR should be avoided unless it fixes the current incompatibility.

File: doc/cprover-manual/cbmc-unwinding.md (hunk 1 above)
Comment (tautschnig) [reply doc/cprover-manual/cbmc-unwinding.md]: Is a bigger rewrite of that sentence on the cards? How about: "The basic idea of CBMC is to model a program's execution up to a bounded number of steps."

Comment (edstenson) [reply doc/cprover-manual/cbmc-unwinding.md]: Thanks, folks. Appreciate your help. I've included that sentence and pushed the file again.

**Review by kroening — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:no_entry_sign:
This PR failed Diffblue compatibility checks (cbmc commit: 80a8f704d8f36162dc933a271e069d34d6b1f0f8).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/93856130
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

The incompatibility may have been introduced by an earlier PR. In that case merging this
PR should be avoided unless it fixes the current incompatibility.

**Pull request merged**